src/ts-serialize.ts

```diff
@@ -31,6 +33,17 @@ export function serializeTsDeclarations(
   return '';
 }
 
+function serializeDocument(node: Document): string {
+  let out = '';
+  for (let i = 0; i < node.members.length; i++) {
```

Comment:
`return node.members.map(serializeTsDeclarations).join('\n')` is shorter and potentially faster (as constructing the outer string is done all at once)

Reply:
Done.

Reply:
Done. Although note it had to be:

`return node.members.map((m) => serializeTsDeclarations(m)).join('\n');`

Because otherwise `map` is passing the current index as the second parameter, which becomes our depth, which makes our indentation wrong. That was a fun one to debug.